migrations/20200824194331-patient-gender.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.bulkUpdate(
+          'patients',
+          { gender: 'F' },
+          { gender: { [Sequelize.Op.regexp]: '.*' } },
+          { transaction: t }
+        ),
```

Comment:
I think this is what we want, right?
```suggestion
        queryInterface.bulkUpdate(
          'patients',
          { gender: 'M' },
          { gender: 'Male' },
          { transaction: t }
        ),
        queryInterface.bulkUpdate(
          'patients',
          { gender: 'F' },
          { gender: 'Female' },
          { transaction: t }
        ),
        queryInterface.bulkUpdate(
          'patients',
          { gender: 'F' },
          { gender: { [Sequelize.Op.regexp]: '.*' } },
          { transaction: t }
        ),
```

Reply:
not quite - i believe the last `bulkUpdate` will just overwrite the first 2, so ideally we be more specific and query for all values that are `NOT 'M' OR 'F'`

additionally, you could do it in this order
- both `Male` -> `M` and `Female` -> `F` first
- then `NOT 'M' OR 'F'`-> `F`
- then change column to use new enum